allennlp/nn/util.py

```diff
@@ -421,6 +423,14 @@ def viterbi_decode(tag_sequence: torch.Tensor,
         other, or those transitions are extremely unlikely. In this situation we log a
         warning, but the responsibility for providing self-consistent evidence ultimately
         lies with the user.
+    allowed_start_transitions : torch.Tensor, optional, (default = None)
+        An optional tensor of shape (num_tags,) describing which tags the START token
+        may transition TO. If provided, additional transition constraints will be used for
```

Comment:
Probably clearer to use `*to*` instead of `TO`.  Same below.

Reply:
done